language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FunctionCallExpressionNodeContext.java

```diff
@@ -84,4 +93,26 @@ public void sort(BallerinaCompletionContext context,
                     .setSortText(SortingUtil.genSortTextByAssignability(completionItem, symbol));
         }
     }
+
+    private List<LSCompletionItem> getNamedArgExpressionCompletionItems(BallerinaCompletionContext context,
+                                                                        FunctionCallExpressionNode node) {
+        List<LSCompletionItem> completionItems = new ArrayList<>();
+        Optional<SemanticModel> semanticModel = context.currentSemanticModel();
+        if (semanticModel.isEmpty()) {
+            return completionItems;
+        }
+        Optional<Symbol> symbol = context.currentSemanticModel().get().symbol(node);
```

Comment:
Check whether the semantic model is empty, before the usage

Reply:
HAve done the check on line 101. But haven't used it. :) Will do